src/apps/filters/pointToCell.c

```diff
@@ -17,7 +17,7 @@
  * @brief stdin/stdout filter that converts from lat/lon coordinates to integer
  * H3 indexes
  *
- *  See `geoToH3 --help` for usage.
+ *  See `pointToCell --help` for usage.
```

Comment:
Should definitely get that decided before we land this.

I would really like to know if anyone actually uses these utility programs, though.

Reply:
Discussed in the steering committee. @isaacbrodsky suggests consolidating these in a single filter command like `h3 pointToCell <...args>`. This would be a separate PR, obviously, so what you have here seems sufficient for now.

Reply:
@dfellis I use these :). I can't be the only one that uses pipes in the unix shell to transform data.

I concur with @nrabinowitz that the "CLI utils is one place where the more generic names aren't helpful". Within H3 itself the decision was made to make lat/lon equivalent to `point`, etc. But the command line user is outside H3 trying to transform between specific location representations, and using the abstract terms obscures that.